components/Header/HeaderItems.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import { usePathname } from 'next/navigation';
+import { LINKS } from '@/config/consts';
+import { NavDropdown } from './NavDropdown';
+import Link from 'next/link';
+
+interface HeaderItemsProps {
+  headerText: string[];
+}
```

Comment:
This is not used, was your intention to replace the any (check below comment) type with  `HeaderItemsProps`?